conda-forge/go-feedstock
# anaconda 1.19

- remove sed as requirement on osx
- disable cgo when building bootstrap go
- Revert "disable cgo when building bootstrap go"
- skip linux-ppc64le
- Add cross-compile capability for more platforms
- Don't retain build cache
- Bump version to 1.11.4
- Drop build number
- Modify go 1.16 recipe for defaults
- update 1.19.1
- reset build-number to zero
- disable linux-s390x
- enable bootstrap on linux-64 too
- disable dependency to 1.4bootstrap go
- disable overlinking
- add binutils for linux archs
- attempt to fix osx-64
- fix test stuff
- adjust osx-64 test for cgo
- adjust windows testing for not failing
- make go tool dist test pass on ppc64le
- adjust osx-64, and ppc64le testing
- update used SDK during build
- enable display of execution steps on testing for unix
- fix windows test file
- undo debugging
- adjust windows testing
- update home/dev/doc urls
- remove no longer used 'go14' dependency
- summarize purpose of patches, and best practice on updating patches to newer version
- add debug print
- re-trigger build
- re-trigger
- trigger
- re-trigger
- make all tests pass on fail
- moved http to https urls
- Updated version to 1.19.3
- Updated SHAs
- MNT: Re-rendered with conda-build 3.23.3, conda-smithy 3.23.0, and conda-forge-pinning 2023.03.06.17.29.54
